src/PatchOrchestrationApplication/NodeAgentNTService/src/Utility/OperationResultFormatter.cs

```diff
@@ -146,6 +148,7 @@ class WUUpdateWrapper
         public IUpdate2 Update { get; private set; }
         public bool IsDownloaded { get; set; }
         public bool IsInstalled { get; set; }
+        public int HResult { get; set; } = 0;// Should be set in a way that it is null also.
```

Comment:
>// Should be set in a way that it is null also. [](start = 45, length = 47)

Didnt get the comment.. How can int be null?

Reply:
It wont be null in case of DataContratSerializer. I will remove this comment